### PR TITLE
Misc scripted/build fixes/improvements

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=develop&zinc=pull/795/head&io=develop&librarymanagement=develop&util=develop&website=develop&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

**Review by eed3si9n — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -503,37 +500,6 @@ lazy val compilerBridge = (projectMatrix in internalPath / "compiler-bridge")
       case (2, y) if y == 11 || y == 12 => new File(scalaSource.value.getPath + "_2.11-12")
       case (2, y) if y >= 13            => new File(scalaSource.value.getPath + "_2.13")
     }.toList),
-    cleanSbtBridge := {
```
Comment (eed3si9n) [inline build.sbt:506]: Currently I'm trying to debug compiler bridge by making changes, and running sbt tasks. Something is caching the compiler it seems, and updates are not sticking.

Comment (dwijnand) [reply build.sbt:506]: It's not going to be related to this deletion, as the values were all wrong here (e.g. sbtScalaVersion).

Comment (eed3si9n) [reply build.sbt:506]: https://github.com/sbt/sbt/issues/5735

**Pull request merged**